Dockerfile.ubi

```diff
@@ -0,0 +1,21 @@
+FROM redhat/ubi8
+
+RUN yum update -y
+RUN yum install iscsi-initiator-utils -y
+RUN yum install kmod -y
+
+
+COPY /seagate-exos-x-csi-controller /usr/local/bin/
+COPY /seagate-exos-x-csi-node /usr/local/bin/
```

Comment:
Best practice is to limit the number of layers by minimizing the number COPY, ADD and RUN commands (each creates a new layer).  Here you could get by with one COPY and one RUN. Not sure it makes much of a difference here.

Reply:
Thanks Chris. Cleaned up and optimized the dockerfile and docker image, plus added labels and Seagate as Author. Use docker inspect ghcr.io/seagate/seagate-exos-x-csi:latest to see the changes.